steward/src/commands/keys/eth/import_from_key.rs

```diff
@@ -0,0 +1,61 @@
+use super::show::ShowKeyCmd;
+use crate::application::APP;
+use abscissa_core::{clap::Parser, Application, Command, Runnable};
+use k256::pkcs8::ToPrivateKey;
+use signatory::FsKeyStore;
+use std::path;
+
+#[derive(Command, Debug, Default, Parser)]
+#[clap(
+    long_about = "DESCRIPTION \n\n Import an external Eth key via private key.\n This command will recover a Eth key, storing it in the keystore. \n It takes a keyname and private key."
+)]
+
+pub struct ImportFromKeyCmd {
+    /// Eth keyname.
+    pub name: String,
+
+    /// Overwrite key with the same name in the keystore when set to true. Takes a Boolean.
+    #[clap(short, long)]
+    pub overwrite: bool,
+
+    /// private-key optional. When absent you'll be prompted to enter it.
+    pub key: Option<String>,
```

Comment:
Should also specify formatting details here.

Reply:
Done!